### DISCUSSION TITLE
Initial running of the template does not run

### DISCUSSION THREAD

**Discussion by ikeronny25** [joaomlourenco/novathesis#261] [Q&A] [answered]
Hello All, may I please know why my initial download of the template.tex file in the zip folder does not run for me to then customize as to what fits my details. I will appreciate the reason why and how to overcome it. Thank you.

**Comment by ikeronny25** [comment]
I downloaded the very current version dated 19-12-2021.

**Comment by joaomlourenco** [comment]
Yes it does run.  Just downloaded it and it “compiles”.
If you want some help please provide clear and detailed information about your environment:
— Operating system
— LaTeX distribution
— LaTeX editor
— How are you trying to "compile" the initial source
— etc

**Reply by ikeronny25** [reply]
Dear Prof, I am using a Mac OS with TeXstudio 4.1.2 (git 4.1.2) Using Qt Version 6.2.2, compiled with Qt 6.2.2 R. After downloading, I try to compile the template.tex but it gives me an error.

**Comment by joaomlourenco** [comment]
Did you install LaTeX as well?  Or just TexStudio?
Please watch these vídeos first…
https://www.youtube.com/playlist?list=PLfNtAAKS03dDRNd6YoTcgbW5qIaM9bE9r

**Reply by ikeronny25** [reply]
Dear Prof,

Kindly find attached the error message I receive after running the initial template downloaded

<img width="1440" alt="Thesis template shot" src="https://user-images.githubusercontent.com/91667327/147695883-ccb18e12-291d-48e4-a353-ede4765abcff.png">
<img width="1440" alt="Thesis template shot" src="https://user-images.githubusercontent.com/91667327/147695973-678a0108-d132-4c3e-9ad9-10c5f2a7e22b.png">

**Comment by joaomlourenco** [comment]
You did not answer my questions…  

Did you install LaTeX as well? Which LaTeX distribution?
Please watch these videos first…
https://www.youtube.com/playlist?list=PLfNtAAKS03dDRNd6YoTcgbW5qIaM9bE9r

**Reply by ikeronny25** [reply]
Yes, Prof. I did install MacTeX.

**Comment by joaomlourenco** [comment]
Please open a terminal (cmd+space: terminal) and run the following commands:
```
cd /Users/mac/Downloads/joaomlourenco-novathesis-ec70f07/
make
```
and then copy&paste the output here.

**Reply by joaomlourenco** [reply]
Even better,  please **download the latest version of the template**, then
```
cd /Users/mac/Downloads/YOUR_NEW_TEMPLATE_FOLDER
make
```
and then copy&paste the output here.

**Reply by ikeronny25** [reply]
![Uploading Untitled 2.png…]()

**Comment by ikeronny25** [comment]
<img width="1440" alt="Untitled 2" src="https://user-images.githubusercontent.com/91667327/147765935-36ac2af4-5c06-49a9-a731-583927ad5358.png">

**Comment by joaomlourenco** [comment]
Dear @ikeronny25,

If you don't do what I ask you to do, I can't help you.  I'm sorry…
I need you to download the latest version of the template, then open a terminal and
```
cd ~/Downloads/YOUR_NEW_TEMPLATE_FOLDER
make
```
and then copy&paste the output here.

**Reply by ikeronny25** [reply]
(base) MacBook-Pro:~ mac$ cd ~/Downloads/joaomlourenco-novathesis-c217e0e
(base) MacBook-Pro:joaomlourenco-novathesis-c217e0e mac$ make
latexmk -time -shell-escape -synctex=1  -pdf -silent template
Accumulated processing time = 0.02
(base) MacBook-Pro:joaomlourenco-novathesis-c217e0e mac$

**Reply by ikeronny25** [reply]
Sorry, initially, i did not realize you wanted the output of the terminal

**Reply by joaomlourenco** [reply]
Please do a
```
make clean
make
```
and copy the output please.

**Comment by ikeronny25** [comment]
(base) MacBook-Pro:joaomlourenco-novathesis-c217e0e mac$ make clean
Latexmk: This is Latexmk, John Collins, 17 Jan. 2018, version: 4.55.
Accumulated processing time = 0.02
(base) MacBook-Pro:joaomlourenco-novathesis-c217e0e mac$ make
latexmk -time -shell-escape -synctex=1  -pdf -silent template
Latexmk: Run number 1 of rule 'pdflatex'
This is pdfTeX, Version 3.14159265-2.6-1.40.19 (TeX Live 2018) (preloaded format=pdflatex)
 \write18 enabled.
entering extended mode
Latexmk: Log file says no output from latex
Latexmk: For rule 'pdflatex', no output was made
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 1
      Refer to 'template.log' for details
'pdflatex  -shell-escape -synctex=1 -interaction=batchmode -recorder  "template.tex"': time = 0.38
Accumulated processing time = 0.48
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs of latex/pdflatex.
make: *** [Makefile:20: pdf] Error 12
(base) MacBook-Pro:joaomlourenco-novathesis-c217e0e mac$

**Reply by ikeronny25** [reply]
Dear Professor,

Thank you very much for your time and direction. I have installed a new MacTeX and the codes are running perfectly this time. Is there some more configuration I will need to do for the efficient running of the codes?

I humbly would like to bring to your attention that the PhD programme in Statistics and Risk Management at the Department of Mathematics is not on the list of PhD programmes stated in the configuration file 9 for FCT.

Once again, I am grateful.

**Reply by joaomlourenco** [reply]
I took the list from [FCT-NOVA site](https://www.fct.unl.pt/ensino/doutoramentos).  It seems it is outdated.  Thanks for letting me know.    I will add this degree to the list.

And I will add a check on the installed LaTeX version as well…  to avoid this error for future users.

**Comment by joaomlourenco** [comment, marked as answer]
Your LaTeX is too old.  The template won't work with that version.
Please update your LaTeX installation (MacTeX) to the latest version.

**Comment by ikeronny25** [comment]
Dear Prof,

Sorry I forgot, please I do not see the option of the PhD programme showing on the cover page. For instance with the PhD programme in Statistics and Risk Management, we have the Actuarial Science option. I put this in that option but I cannot see it on the cover page.

Thank you.

**Comment by joaomlourenco** [comment]
I will contact the FCT-NOVA designer and ask her how the specialization shall be printed in the cover.  For now… let it be as it is! ::)
